cmake/modules/FindStdFilesystem.cmake

```diff
@@ -20,6 +20,7 @@ macro(try_std_filesystem_library _library _result)
   unset(_std_filesystem_try_compile_arg)
   if(_std_filesystem_compiles)
     set(${_result} ${_library})
+    MESSAGE(STATUS "Found: std_filesystem: ${_library}")
```

Comment:
please drop this line. `find_package_handle_standard_args()` will print out `${StdFilesystem_LIBRARY}` for us. like
```
Found StdFilesystem: stdc++fs
```

Reply:
@tchaikov 
Reason I added that was because I was missing it in my output.
But it is running in Jenkins as we speak, so I can check:
-- Found: std_filesystem: c++experimental
-- Found StdFilesystem: c++experimental  
Will kill this line.
And fix the commit.